### PR TITLE
feat: add nothing

### PR DESCRIPTION
**Pull request by fauortiz**
AB#1234

o_o

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def mouse_drag_events(self, event):
             # If the user is dragging the mouse with the right mouse button
             if event.buttons[2] == 1: pass
 
-
+# test
```
Comment (castilloglenn) [inline main.py:112]: why did you add a comment here

P.S. 😘

File: main.py [hunk 2]
```diff
@@ -135,3 +135,6 @@ def clear_resources(self):
 if __name__ == '__main__':
     game = Game()
     game.run_main()
+
+
+# hello!
```
Comment (castilloglenn) [inline main.py:140]: yeah, why add comment here too

P.S. 😘

File: main.py (hunk 1 above)
Comment (fauortiz) [reply main.py:112]: gusto ko lang

**Review by castilloglenn — APPROVED** [review]
LGTM community 👍🏼🏳️‍🌈
